app/controllers/modeles_controller.rb

```diff
@@ -21,7 +21,7 @@ def show
   end
 
   def new
-    @modele = Modele.new
+    @modele = Modele.new(new_modele_params)
```

Comment:
```suggestion
    @modele = Modele.new(modele_params)
```

Why not using existing `modele_params`?

Reply:
`modele_params` expects a `modele` first, which makes giving parameters to path helpers less "clean", for example:  
`new_modele_path("modele[architecture_id]" => @architecture.id)`

I changed it and we are now using `modele_params`